apps/spruce/src/components/AprilFoolsEvergreenRedesign/index.tsx

```diff
@@ -0,0 +1,181 @@
+import { useState, forwardRef, useImperativeHandle, useEffect } from "react";
+import { css, Global } from "@emotion/react";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import Cookies from "js-cookie";
+import { zIndex } from "@evg-ui/lib/constants/tokens";
+import { SEEN_COMIC_SANS_PRANK } from "constants/cookies";
+
+export interface EvergreenRedesignModalHandle {
+  openModal: () => void;
+}
+
+const EvergreenRedesignModal = forwardRef<EvergreenRedesignModalHandle>(
+  (_, ref) => {
+    const [modalOpen, setModalOpen] = useState(
+      Cookies.get(SEEN_COMIC_SANS_PRANK) !== "true",
+    );
+    const [comicSansEnabled, setComicSansEnabled] = useState(false);
+    const [isFollowUp, setIsFollowUp] = useState(false);
+
+    useImperativeHandle(ref, () => ({
+      openModal: () => {
+        setModalOpen(true);
+        setIsFollowUp(false);
+      },
+    }));
+
+    const [isFolding, setIsFolding] = useState(false);
+
+    const triggerComicSansWithAnimation = () => {
+      setIsFolding(true);
+      setModalOpen(false); // Hide original modal
+
+      setTimeout(() => {
+        setComicSansEnabled(true);
+        setIsFolding(false);
+
+        // Now start the 10s timer for the follow-up prank modal
+        setTimeout(() => {
+          setIsFollowUp(true);
+          setModalOpen(true);
+          Cookies.set(SEEN_COMIC_SANS_PRANK, "true", { expires: 2 });
+        }, 10000);
+      }, 2000); // Match animation duration
```

Comment:
would recommend shortening duration to 500 or under

Reply:
Removed it all together since it didn't have the desired effect